### ISSUE TITLE
Webpack Newbie: How can I buildSchema() with a glob resolvers ?

### ISSUE THREAD

**Issue by kktos** [19majkel94/type-graphql#238]
Hi there,
I've a newbie question, sorry about that....

I was trying to follow the doc with a simple
`	const schema = await buildSchema({
		resolvers: [ __dirname + "/db/**/*.resolver.ts"]
});`

And it's obviously not working as I'm building the thing with Webpack.
I've initially thought of __dirname and change my settings accordingly but that's not the problem.
Told you, webpack newbie here.
The problem is quite simple, webpack has no knowledge of the files I want to dynamically include....
I'm pretty sure it is a very well known problem.
Anyone has a solution for me ?
Kudos in advance.
/thierry

**Comment by 19majkel94** [comment]
Github issues are not a helpdesk - they're for tracking bug reports, feature requests and discussions.

Please ask the question on gitter chat where the community might help you:
https://gitter.im/type-graphql/Lobby

**Comment by 19majkel94** [comment]
> Told you, webpack newbie here.

That's the problem, why not ask on webpack issues or stackoverflow?
It's not a TypeGraphQL issue or bug.

> webpack has no knowledge of the files I want to dynamically include....

So add the knowledge by pointing all the resolvers file in the entry section:
https://github.com/webpack/webpack/issues/370

**Comment by kktos** [comment]
I won't argue with you as you're totally right, it is not a typegraphql problem and you didn't write webpack.
ok, that being said, I thought my question could be a bit relevant nevertheless as webpack is one of the most used packager so we all need, more or less, to deal with it.
Therefore, i thought someone here has already encountered such a problem and had a solution to share.
Do you get my point ?
I wasn't complaining at all, just asking nicely.
So if you ever find a nice solution, feel free to add it to your doc as I humbly believe it could help some of us.
Thanks a lot and peace, my friend, sincerely,
/Thierry

**Comment by 19majkel94** [comment]
> i thought someone here has already encountered such a problem and had a solution to share.

If you didn't found anything using Github search and it's not located in [the FAQ](https://19majkel94.github.io/type-graphql/docs/faq.html) - then no, it's not a problem that someone had.

> Do you get my point?

Of course but, as I said, GitHub issues is not a helpdesk. All the community is gathered in Gitter chat when you can freely ask your question and there are people online that can help you. People doesn't look into TypeGraphQL issues on GitHub so often so they won't see the opportunity to help you.

I've even created a issues templates for that case:

![image](https://user-images.githubusercontent.com/10618781/51790246-ba891a80-2192-11e9-9dbc-ff817e12fa20.png)

With a large warning - "Please don't post issues with questions here, use Gitter or StackOverflow. Thanks!". 

Without a minimal amount of rules, there wouldn't be here 41 opened issues but 141. And I'm here alone, spending my free time on open source, so answering n00bish questions instead focusing on the code and the development is not the thing I want to do.

So to sum up: I've closed your issue because it's webpack related (not about TypeGraphQL), it's not a bug report or feature request but a n00bish question and it's a question that should be asked in Gitter chat or Stackoverflow.
